test/interpreter/function-varp.spec.ts

```diff
@@ -0,0 +1,22 @@
+import {ErrorType, HyperFormula} from '../../src'
+import {adr, detailedError} from '../testUtils'
+import {ErrorMessage} from '../../src/error-message'
+
+describe('Function VAR.P', () => {
+  it('should take at least one argument', () => {
+    const engine = HyperFormula.buildFromArray([
+      ['=VAR.P()'],
+    ])
+
+    expect(engine.getCellValue(adr('A1'))).toEqual(detailedError(ErrorType.NA, ErrorMessage.WrongArgNumber))
+  })
+
+  it('should calculate variance (population)', () => {
+    const engine = HyperFormula.buildFromArray([
+      ['=VAR.P(2, 3)'],
+      ['=VAR.P(B2:D2, E2, F2)', 2, 3, 'foo', 'bar', 4],
+    ])
+    expect(engine.getCellValue(adr('A1'))).toEqual(0.25)
+    expect(engine.getCellValue(adr('A2'))).toBeCloseTo(2/3, 6)
+  })
+})
```

Comment:
Some tests missing:

- If only one number is provided, returns 0.
- Logical values, and text representations of numbers that you type directly into the list of arguments are counted.
- If an argument is an array or reference, only numbers in that array or reference are counted. Empty cells, logical values, text, or error values in the array or reference are ignored.
- Arguments that are error values or text that cannot be translated into numbers cause errors.
- If you want to include logical values and text representations of numbers in a reference as part of the calculation, use the VARPA function.

Source: 
https://support.microsoft.com/en-us/office/var-p-function-73d1285c-108c-4843-ba5d-a51f90656f3a
https://docs.oasis-open.org/office/OpenDocument/v1.3/csprd02/part4-formula/OpenDocument-v1.3-csprd02-part4-formula.html#__RefHeading__1018790_715980110